src/test/scala/dx/exec/TaskRunnerTest.scala

```diff
@@ -127,16 +126,16 @@ class TaskRunnerTest extends AnyFlatSpec with Matchers {
       try {
         val outputsFile = pathFromBasename(s"${wdlName}_output.json")
         assert(Files.exists(outputsFile))
-        Some(SysUtils.readFileContent(outputsFile).parseJson.asJsObject.fields)
+        Some(Util.readFileContent(outputsFile).parseJson.asJsObject.fields)
       } catch {
         case _: Throwable =>
           None
       }
 
     // Create a clean directory in "/tmp" for the task to use
     val jobHomeDir: Path = Files.createTempDirectory("dxwdl_applet_test")
-    SysUtils.deleteRecursive(jobHomeDir.toFile)
-    SysUtils.safeMkdir(jobHomeDir)
+    Util.deleteRecursive(jobHomeDir)
+    Util.createDirectories(jobHomeDir)
```

Comment:
Not sure in which cases we need these two methods in addition to createTempDirectory() and in which we don't. I made a comment (a question) about it in Pr #429

Reply:
right - these are artifacts. I've removed them.